packages/ng/option/src/lib/item/tree-option-item.component.scss

```diff
@@ -1,198 +1,194 @@
-//       left: 1.5rem;
-//     }
-
-//     &::after {
-//       left: 1.5rem;
-//     }
-//   }
-// }
-
-// :host-context(.lu-picker-panel.mod-multiple lu-tree-option) {
-//   .optionItem-value {
-//     padding-left: 3rem;
-//   }
-// }
-
-// :host-context(.lu-picker-panel lu-tree-option lu-tree-option) {
-//   .optionItem-value {
-//     padding-left: 2.5rem;
-
-//     &::before {
-//       left: 2.5rem;
-//     }
-
-//     &::after {
-//       left: 2.5rem;
-//     }
-//   }
-// }
-
-// :host-context(.lu-picker-panel.mod-multiple lu-tree-option lu-tree-option) {
-//   .optionItem-value {
-//     padding-left: 4rem;
-//   }
-// }
-
-// :host-context(.lu-picker-panel lu-tree-option lu-tree-option lu-tree-option) {
-//   .optionItem-value {
-//     padding-left: 3.5rem;
-
-//     &::before {
-//       left: 3.5rem;
-//     }
-
-//     &::after {
-//       left: 3.5rem;
-//     }
-//   }
-// }
-
-// :host-context(.lu-picker-panel.mod-multiple lu-tree-option lu-tree-option lu-tree-option) {
-//   .optionItem-value {
-//     padding-left: 5rem;
-//   }
-// }
-
-// :host-context(.lu-picker-panel lu-tree-option lu-tree-option lu-tree-option lu-tree-option) {
-//   .optionItem-value {
-//     padding-left: 4.5rem;
-
-//     &::before {
-//       left: 4.5rem;
-//     }
-
-//     &::after {
-//       left: 4.5rem;
-//     }
-//   }
-// }
-
-// :host-context(.lu-picker-panel.mod-multiple lu-tree-option lu-tree-option lu-tree-option lu-tree-option) {
-//   .optionItem-value {
-//     padding-left: 6rem;
-//   }
-// }
-
-// :host-context(.lu-picker-panel lu-tree-option lu-tree-option lu-tree-option lu-tree-option lu-tree-option) {
-//   .optionItem-value {
-//     padding-left: 5.5rem;
-
-//     &::before {
-//       left: 5.5rem;
-//     }
-
-//     &::after {
-//       left: 5.5rem;
-//     }
-//   }
-// }
-
-// :host-context(.lu-picker-panel.mod-multiple lu-tree-option lu-tree-option lu-tree-option lu-tree-option lu-tree-option) {
-//   .optionItem-value {
-//     padding-left: 7rem;
-//   }
-// }
-
-// :host-context(.lu-picker-panel lu-tree-option lu-tree-option lu-tree-option lu-tree-option lu-tree-option lu-tree-option) {
-//   .optionItem-value {
-//     padding-left: 6.5rem;
-
-//     &::before {
-//       left: 6.5rem;
-//     }
-
-//     &::after {
-//       left: 6.5rem;
-//     }
-//   }
-// }
-
-// :host-context(.lu-picker-panel.mod-multiple lu-tree-option lu-tree-option lu-tree-option lu-tree-option lu-tree-option lu-tree-option) {
-//   .optionItem-value {
-//     padding-left: 8rem;
-//   }
-// }
-// /* clean-css ignore:end */
+@import "_definitions";
+@include optionItemStyle;
+
+.optionItem {
+  &:hover {
+    .optionItem-value {
+			background: var(--palettes-grey-50);
+    }
+
+    .optionItem-icons {
+      opacity: 1;
+    }
+  }
+}
+
+.optionItem-value {
+  padding: var(--components-options-item-padding-vertical) var(--components-options-item-padding-horizontal) var(--components-options-item-padding-vertical) .5rem;
```

Comment:
`padding: var(--components-options-item-padding-vertical) var(--components-options-item-padding-horizontal);` ?

Reply:
Indeed. I'm not sure why it has a specific style like this. Maybe Option item horizontal padding was larger in the past 🤔